newIDE/app/src/AssetStore/AssetCard.js

```diff
@@ -99,7 +100,7 @@ export const AssetCard = ({ id, assetShortHeader, size }: Props) => {
         }}
       >
         <Text noMargin style={styles.title} color="inherit">
-          {assetShortHeader.name}
+          {capitalize(assetShortHeader.name)}
```

Comment:
I wonder if we should use CSS instead in order to reduce the JS workload. But I'm not sure CSS is faster to make this change.
For AssetDetails, it would be only one string to change so it's not important to me. Here there are more items.